do.py

```diff
@@ -1178,6 +1182,54 @@ def authors(ctx_obj, revision_args):
     except subprocess.CalledProcessError:
         print('Error caught: Incorrect revision start or revision end')
 
+"""
+- Currently the run-job/reuse fails (issue https://github.com/scipy/scipy/issues/16009)
+"""
+@cli.cls_cmd('act')
+class Act(Task):
+    """:flashlight: A tool which provides a handy way to run GitHub Actions locally using Docker
+
+    Installation guide :arrow_down:
+    - Install docker desktop/toolkit
+    - Install act
+        * Using homebrew: brew install act
+        * Using bashscript: curl https://raw.githubusercontent.com/nektos/act/master/install.sh | sudo bash
+
+    Options available:
+        * list (lists all the workflows present)
+        * run-job (runs a particular job)
```

Comment:
We should either talk about `job` or `workflow` to have a coherent wording.

Reply:
Thanks for pointing this. I will make the tweak to `workflow/workflows`